Sources/JWTKeychain/Routes/Api/ApiUserRoutes.swift

```diff
@@ -34,7 +34,7 @@ public struct ApiUserRoutes: RouteCollection {
         // Auth routes
         path.post(handler: controller.register)
         path.post("login", handler: controller.logIn)
-        path.post("reset-password", "request", handler: controller.resetPasswordEmail)
+        path.get("reset-password", "request", handler: controller.resetPasswordEmail)
```

Comment:
This is for "creating" my request to reset a pw, right? Reasons for changing to `GET`?